src/api/goods.ts

```diff
@@ -1,19 +1,44 @@
 import { Good } from '../types/Good';
 
+function sortByName<T extends Good>(value: T[]): T[] {
+  return [...value].sort((prev, curr) => prev.name.localeCompare(curr.name));
+}
+
+function sliceByNumber(value: Good[], number: number): Good[] {
```

Comment:
When variable contains an array inside it's better to put name in plural, like `values` instead of `value`